docs/varnish.md

```diff
@@ -49,3 +20,21 @@ The Ops Agent collects the following metrics from your varnish instance.
 | workload.googleapis.com/varnish.client.request.count       | cumulative | {requests}    | cache_name, state                   | The client request count                                         |
 | workload.googleapis.com/varnish.client.request.error.count | cumulative | {requests}    | cache_name, http.status_code        | The client requests errors received by status code.              |
 | workload.googleapis.com/varnish.backend.request.count      | cumulative | {requests}    | cache_name                          | The backend requests count                                       |
+
+## Logs
+
+Varnish logs contain the [`httpRequest` field](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#httprequest):
+
+| Field                       | Type                                                                                                                            | Description                                                   |
+|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------|
+| `httpRequest.protocol`      | string                                                                                                                          | Protocol used for the request                                 |
+| `httpRequest.referer`       | string                                                                                                                          | Contents of the `Referer` header                              |
+| `httpRequest.remoteIp`      | string                                                                                                                          | Client IP address                                             |
```

Comment:
Are we missing `httpRequest.serverIp` from the scope doc?

Reply:
no, see above. I will update the scope doc accordingly.